src/magic_folder/snapshot.py

```diff
@@ -210,6 +324,124 @@ def deserialize_dict(snapshot_dict, author):
 
         return deserialize_dict(local_snapshot_dict, author)
 
+@attr.s
+class RemoteSnapshot(object):
+    """
+    Represents a snapshot corresponding to a particular version of a
+    file authored by a particular human.
+
+    :ivar name: the name of this Snapshot. This is a mangled path
+        relative to our local magic-folder path.
+
+    :ivar metadata: a dict containing metadata about this Snapshot.
+
+    :ivar parents_raw: list of capablitiy-strings instances of our
+        parents
```

Comment:
"Whatever tahoe accepts"? Tahoe's code says it must be `str` so I guess that means "bytes" in python3 land? Or should they be "unicode" because users sometimes supply them (not for this list). Really, they're ASCII-only strings (base32 + "URI:DIR:" etc prefixes)?